TTMouseclickSimulator/SampleProjects/ToontownRewritten/Auto Fishing - Estate.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<SimulatorProject xmlns="https://github.com/TTExtensions/MouseClickSimulator">
+  <Title>Automatic Fishing in Toon Estate</Title>
+  <Description>
+The Toon will automatically fish in your estate.
+Before you click on Start, make sure that
+• your fish bucket is empty,
+• you have enough JellyBeans for 20 casts,
+• your toon is standing on the board nearest to the bucket
+Note: Doodles may cause the simulator to fail.
+  </Description>
+
+  <MainAction>
+
+    <Compound type="Sequential">
+
+      <!-- 
+	  		Use the automatic fishing action for estate 20 times,
+       		and ensure Toon remains fishing instead of going to bed.
+      -->
+      <Loop count="5"> 
+        <AutomaticFishing scan1="248, 239" scan2="1492, 660" bubbleColorRgb="49, 128, 120"
```

Comment:
I noticed that the taget Y position of the fishing lure is too high, probably because the estate fishing uses a different perspective. This can be improved by setting the `scanResultYAdjustment` attribut, for example (I also improved the scan coordinates and colors a bit):
```xml
        <AutomaticFishing scan1="200, 150" scan2="1492, 660" scanResultYAdjustment="35"
                          bubbleColorRgb="56, 129, 122" toleranceRgb="7, 5, 5" />
```

Reply:
looks good!